src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -998,5 +1007,32 @@ public LastRunInfo GetlastRunInfo()
             return lastRunInfo;
         }
 
+        private IntPtr GetClientRef()
+        {
+            return CallNativeObjectMethodRef("getClient", "()Lcom/bugsnag/android/Client;", new object[] { });
+        }
+
+        public void AddFeatureFlag(string name, string varient)
+        {
+            var methodId = AndroidJNI.GetMethodID(AndroidJNI.FindClass("com/bugsnag/android/Client"), "addFeatureFlag", "(Ljava/lang/String;Ljava/lang/String;)V");
```

Comment:
Is getting the JNI method and finding the class slow? Do we need to cache the methodId?

Reply:
These looks aren't cheap, but there's also a question on how frequently they are called (up-front caching might not be worth it)?

That being said: caching them up-front also keeps things consistent with the other functions we call?

Reply:
Cached 👍